c/models/aggregator.cc

```diff
@@ -730,7 +750,7 @@ void Aggregator<T>::group_nd() {
   if (do_projection) pmatrix = generate_pmatrix(ncols);
 
   // Figuring out how many threads to use.
-  size_t nth0 = get_nthreads(nrows);
+  size_t nth = std::min(get_nthreads(nrows), dt::get_num_threads());
```

Comment:
`get_nthreads()` uses `config::nthreads` to determine number of threads, and then did `#pragma omp parallel num_threads(nth0)`. Since now `dt::parallel_region` doesn't accept number of threads as a parameter, how can we change number of threads?

Reply:
i'll add that as a parameter